tldw_Server_API/tests/Chunking/test_xml_tail_preservation.py

```
@@ -10,7 +10,7 @@ def test_xml_chunk_preserves_tail_text():
     chunks = strategy.chunk(xml_text, max_size=50, overlap=0, output_format="xml")
 
     assert chunks
-    root = ET.fromstring(chunks[0])
+    root = ET.fromstring(chunks[0])  # nosec B314
```

Ruff (0.15.1)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)